docs/DeveloperGuide.md

```diff
@@ -412,6 +412,186 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   Use case ends.
 
 
+**Use case: UC05 - Create a group**
+
+**MSS**
+
+1. User requests to list persons
+2. CFG shows a list of relevant persons
+3. User requests to create a group, with specified persons in the list as members
+4. CFG successfully creates the group
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. User could also <ins>search for persons by name/tag/role... etc. (UC01)</ins>
+
+  Use case resumes at step 2.
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. A group with the same name already exists.
+  * 3a1. CFG shows an error message telling the user that the group already exists.
+
+    Use case ends.
+
+
+**Use case: UC06 - List groups**
+
+**MSS**
+
+1. User requests to see all existing groups
+2. CFG shows a list of all existing groups
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. There are no existing groups.
+  * 2a1. CFG shows an empty list.
+
+    Use case ends.
+
+
+**Use case: UC07 - View members of a group**
+
+**MSS**
+
+1. User requests to <ins>see a list of groups (UC06)</ins>
+2. User requests to see members of a group
+3. CFG lists all the members of the group
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. User may not require a list of groups.
+  
+    Use case resumes at step 2.
+
+* 2a. The group with specified name does not exist.
+  * 2a1. CFG shows an error message.
+
+    Use case ends.
+* 3b. The group currently has no members.
```

Comment:
Is this a formatting error

Reply:
Nope, it comes out fine in the actual document! Markdown cares more about the indentation than the newline.

Reply:
shesh ok